test/malli/core_test.cljc

```diff
@@ -997,6 +997,18 @@
                                                (mt/key-transformer
                                                 {:decode #(-> % name (str "_key") keyword)}))))
 
+      (testing "JSON transformer decodes map schema keys"
+        (let [schema [:map
+                      [:a :uuid]
```

Comment:
I think this is a breaking change for users who want to keep keys as strings. Could you add a test case that demonstrates what happens with a schema with string keys?

Reply:
Added an example with `"string"` key in `[:map` in https://github.com/metosin/malli/pull/1135/commits/761d62f6fd66c0c52b7c3825c03186ac0665f2b1.